dvc/remote/gdrive.py

```diff
@@ -153,14 +179,28 @@ def _validate_config(self):
             )
 
         # Validate OAuth 2.0 Client ID configuration
-        if not self._use_service_account and (
-            not self._client_id or not self._client_secret
-        ):
-            raise DvcException(
-                "Please specify Google Drive's client id and "
-                "secret in DVC config. Learn more at "
-                "{}.".format(format_link("https://man.dvc.org/remote/modify"))
-            )
+        if not self._use_service_account:
+            if not self._client_id or not self._client_secret:
```

Comment:
the both are set now, right? remove this check?

Reply:
check is changed to "they both must be set only if one them is set"